### PR TITLE
Handle multiple contacts in interaction CSV export

### PR DESCRIPTION
**Pull request by reupen**
### Description of change

This adds a Contacts column to the interaction CSV export, and removes the previous Contact column.

### Checklist

* [x] Has a new newsfragment been created? Check [changelog/README.rst](https://github.com/uktrade/data-hub-leeloo/blob/master/changelog/README.rst) for instructions
* [ ] Have any relevant search models been updated?
* [ ] Have any relevant fixtures (`fixtures/test_data.yaml`) been updated?
* [ ] Have any relevant select-/prefetch-related field lists in the views and search apps been updated?
* [ ] Has the admin site been updated (for new models, fields etc.)?
* [ ] Has the README been updated (if needed)?

### CODE REVIEW

File: datahub/search/interaction/tests/test_views.py [hunk 1]
```diff
@@ -949,4 +953,21 @@ def test_interaction_export(
...
+
+def _format_actual_csv_row(row):
+    return {key: _format_actual_csv_value(key, value) for key, value in row.items()}
+
+
+def _format_actual_csv_value(key, value):
+    """
+    Sorts the value of 'Contacts' for a row alphabetically as they are unordered at present.
+
+    TODO Django 2.2 adds ordering support to StringAgg, which will remove the need for this as we
```
Comment (elcct) [inline datahub/search/interaction/tests/test_views.py]: I think `TODO:` would look better.

File: datahub/core/test_utils.py [hunk 2]
```diff
@@ -314,6 +314,17 @@ def _map_field(field):
         return 'id' if field == 'pk' else field
 
 
+def join_values(iterable, attr='name', separator=', '):
```
Comment (marcofucci) [inline datahub/core/test_utils.py]: `join_attrs_values` or similar would be better as `join_values` is too generic.

**Comment by reupen** [comment]
Actually this is a little bit more complicated as job title is a column as well, setting this back to 'work in progress'.

**Comment by marcuspp** [comment]
@reupen looks good to me. :+1: I see you are removing columns in the csv download. Does this need to be communicated to other stake holders that may rely on the csv download prior to the change going live?

**Comment by reupen** [comment]
> @reupen looks good to me. +1 I see you are removing columns in the csv download. Does this need to be communicated to other stake holders that may rely on the csv download prior to the change going live?

Ideally, but I'm not sure there is much of a choice here anyway. I'll mention your concern to the rest of the team.

**Pull request merged**